### PR TITLE
Tooltip

### PR DESCRIPTION
**Pull request by mihai-macaneata**

### CODE REVIEW

File: assets/js/components/Beneficiaries.vue [hunk 1]
```diff
@@ -332,6 +390,45 @@ export default Vue.extend({
         .transition(t_)
         .call(this.renderItems);
 
+
+      // tooltip
+      let tip = d3.tip()
+        .attr('class', 'd3-tip benef')
+        .html(function(d){
+         return "<div class='title-container'><img src=/assets/imgs/"
+              + d3.select(this).select('use').attr('href').substring(1) + ".png"
```
Comment (melish) [inline assets/js/components/Beneficiaries.vue]: Extract as a separate variable, before return

File: assets/js/components/Beneficiaries.vue [hunk 2]
```diff
@@ -69,6 +69,66 @@
   }
 }
 
+.d3-tip.benef {
```
Comment (melish) [inline assets/js/components/Beneficiaries.vue]: Please extract the CSS from Beneficiaries.vue in its own resource, we will probably reuse it for the other components

Comment (xlotlu) [inline assets/js/components/Beneficiaries.vue]: WithTooltip mixin or somesuch ...

File: assets/js/components/Beneficiaries.vue [hunk 3]
```diff
@@ -332,6 +390,45 @@ export default Vue.extend({
         .transition(t_)
         .call(this.renderItems);
 
+
+      // tooltip
+      let tip = d3.tip()
+        .attr('class', 'd3-tip benef')
+        .html(function(d){
+         return "<div class='title-container'><img src=/assets/imgs/"
```
Comment (xlotlu) [inline assets/js/components/Beneficiaries.vue]: a) there is specific css ready-made for country flags.
b) you have the country in `d`. d3.select(this).select('use').attr('href').substring(1) is ... not right.

File: assets/js/components/Beneficiaries.vue (hunk 1 above)
Comment (melish) [inline assets/js/components/Beneficiaries.vue]: how about get_flag_name(d.id) ?

**Pull request closed without merging**